docs/modeling-your-data/modeling-your-data-with-dbt/dbt-operation/index.md

```diff
@@ -19,42 +18,44 @@ On this page, `<package>` can be one of: `web`, `mobile`, `ecommerce,` `normaliz
 
 ```bash
 dbt run --selector snowplow_<package>
 ```
 
-## YAML Selectors
-
 Within the packages we have provided a suite of suggested selectors to run and test the models within the packages. This leverages dbt's [selector flag](https://docs.getdbt.com/reference/node-selection/syntax).
 
 <Tabs groupId="dbt-packages">
 <TabItem value="web" label="Snowplow Web" default>
 
-- `snowplow_web`: Recommended way to run the package. This selection includes all models within the Snowplow Web as well as any custom models you have created.
-- `snowplow_web_lean_tests`: Recommended way to test the models within the package. See the testing section for more details.
+- `snowplow_web`: Recommended way to run the package. This selection includes all models within the Snowplow Web as well as any custom models you have created
+- `snowplow_web_lean_tests`: Recommended way to test the models within the package. See the testing section for more details
 
 </TabItem>
 <TabItem value="mobile" label="Snowplow Mobile">
 
-- `snowplow_mobile`: Recommended way to run the package. This selection includes all models within the Snowplow Mobile as well as any custom models you have created.
-- `snowplow_mobile_lean_tests`: Recommended way to test the models within the package. See the testing section for more details.
+- `snowplow_mobile`: Recommended way to run the package. This selection includes all models within the Snowplow Mobile as well as any custom models you have created
+- `snowplow_mobile_lean_tests`: Recommended way to test the models within the package. See the testing section for more details
 
 </TabItem>
 <TabItem value="media" label="Snowplow Media Player">
 
-- `snowplow_web`:  Recommended way to run the package. This selection includes all models within the Snowplow Web and Snowplow Media Player as well as any custom models you have created.
+- `snowplow_web`:  Recommended way to run the package. This selection includes all models within the Snowplow Web and Snowplow Media Player as well as any custom models you have created
 - `snowplow_web_lean_and_media_player_tests`: Recommended way to test the models within the package. See the testing section for more details.
-- `snowplow_media_player_tests`: Runs all tests within the Snowplow Media Player Package and any custom models tagged with `snowplow_media_player`.
-- `snowplow_web_and_media_player_tests`: Runs all tests within the Snowplow Web and Snowplow Media Player Package and any custom models tagged with `snowplow_media_player` or `snowplow_web_incremental`.
+- `snowplow_media_player_tests`: Runs all tests within the Snowplow Media Player Package and any custom models tagged with `snowplow_media_player`
+- `snowplow_web_and_media_player_tests`: Runs all tests within the Snowplow Web and Snowplow Media Player Package and any custom models tagged with `snowplow_media_player` or `snowplow_web_incremental`
 
 </TabItem>
 <TabItem value="normalize" label="Snowplow Normalize">
```

Comment:
For some reason I can't suggest a change or comment in the lines below but you missed a trailing period for Normalize